pulp/pulp.py

```diff
@@ -1901,7 +1901,14 @@ def assignStatus(self, status, sol_status=None):
         :param sol_status: code for the status of the solution
         :return:
         """
-        # TODO: check if status are valid status codes
+        if status not in const.LpStatus:
```

Comment:
I don't think that printing a message is a good idea. I would raise an error or a warning. see the examples with warnings `warnings.warn("Overwriting previously set objective.")` or errors `raise const.PulpError("Duplicated names found in variables:\nto export the model, variable names need to be unique")` in this same file. What do you think?

Reply:
I thought I'll modify to `const.PulpError()`, since it is more like an error than a dangerous behavior. Thanks for your comments!